pmd-core/src/test/java/net/sourceforge/pmd/properties/NonRuleWithAllPropertyTypes.java

```diff
@@ -37,8 +37,8 @@
     public static final StringMultiProperty MULTI_STR = new StringMultiProperty("multiStr", "Multiple string values",
```

Comment:
Btw this class probably doesn't belong in PMD. It says it's there to test UIs, but arguably UIs can use their own test fixtures, and this in in src/test so probably not even shipped anywhere.

Reply:
Yes, if the class is needed in pmd-eclipse, it should be moved there. It's quite an effort to make at available there - as outlined in the class comments. But that's only, because it's in pmd and not in pmd-eclipse...
We should simply delete this file (and the test class PropertyAccessorTest, where it is used).
I can do this, when merging.